library/packages/src/modules/Product.rb

```diff
@@ -198,7 +198,7 @@ def load_zypp
       if use_installed_products?
```

Comment:
live_installation mode is not used anymore, isn't it?, thus, this method will check if  `!Stage.initial` and in the else there is again a !Stage.initial, so looks like we could do something like 

```ruby
PackageSystem.EnsureTargetInit if Stage.Initial
```

Reply:
Lets keep it simple for the RC1...